### ISSUE TITLE
Document #npmignore comment and npmignore utility usage in `.npmignore` file

### ISSUE THREAD

**Issue by jamestalmage** [ljharb/npmignore#2]
File should look like this

```
stuff ignored by .gitignore
...

# npmignore - content above this line is automatically generated and modifications may be omitted
# see npmjs.com/npmignore for more details.
```

**Comment by jonschlinkert** [comment]
makes sense, want to do a pr?

**Comment by jamestalmage** [comment]
sure

**Comment by jamestalmage** [comment]
is my wording good?

**Comment by jamestalmage** [comment]
closed via #3

**Comment by jonschlinkert** [comment]
> is my wording good?

it was great, thanks!
